src/views/Dashboard/Customer/Home/GeneticData/Add.vue

```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
+
+</template>
```

Comment:
Wait is this a draft PR? Why is this open?

Reply:
Initially this PR is only for add genetic data to navigation drawer, I forgot to exclude this page. However, since I finished the display, I have pushed another commit to the PR, please kindly review ya te @agustinustheo . thank you